flutter/shell/platform/tizen/channels/keyboard_channel.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_KEYBOARD_CHANNEL_H_
+#define EMBEDDER_KEYBOARD_CHANNEL_H_
+
+#include <map>
+#include <memory>
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/binary_messenger.h"
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/encodable_value.h"
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/method_channel.h"
+
+namespace flutter {
+
+// Channel to set the system's cursor type.
+class KeyboardChannel {
+ public:
+  explicit KeyboardChannel(BinaryMessenger* messenger);
+  virtual ~KeyboardChannel();
+
+  void UpdatePressingState(uint64_t physical_key,
+                           uint64_t logical_key,
+                           bool is_down);
+
+ private:
+  // Called when a method is called on |channel_|;
+  void HandleMethodCall(
+      const flutter::MethodCall<EncodableValue>& method_call,
+      std::unique_ptr<flutter::MethodResult<EncodableValue>> result);
+
+  // The MethodChannel used for communication with the Flutter engine.
+  std::unique_ptr<flutter::MethodChannel<EncodableValue>> channel_;
+
+  std::map<uint64_t, uint64_t> pressing_records_;
```

Comment:
`pressing_records_` are already being tracked by the `KeyEventChannel`. Can we reuse the values? Please let me know if you want to discuss how to redesign or refactor the code.

Reply:
I updated KeyEventChannel to KeyboardChannel class and added keyboard method channel code